pom.xml

```diff
@@ -3,7 +3,7 @@
 
     <groupId>net.wasdev.wlp.tracer</groupId>
     <artifactId>liberty-opentracing-zipkintracer</artifactId>
-    <version>2.0.2-SNAPSHOT</version>
+    <version>3.0-RC1</version>
```

Comment:
Please change it to `3.0-SNAPSHOT` and the maven release plugin will update this to `3.0-RC1` later.